xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -130,7 +130,7 @@
 #define ADDON_INSTANCE_VERSION_PERIPHERAL_DEPENDS     "addon-instance/Peripheral.h" \
                                                       "addon-instance/PeripheralUtils.h"
 
-#define ADDON_INSTANCE_VERSION_PVR                    "8.0.1"
+#define ADDON_INSTANCE_VERSION_PVR                    "8.0.2"
```

Comment:
You need to increase both version defines, min and „non-min“ both must be 8.0.2, because of the new PVR_CODEC enum value inserted in between existing values.

Reply:
understood, will correct. Thanks!